fe/model_rabfe.py

```diff
@@ -151,38 +151,25 @@ def predict(self,
 
         dG, dG_err, results = estimator_abfe.deltaG(model, sys_params)
 
-        # uncomment if we want to visualize
+        # Save out the pdb
         combined_topology = model_utils.generate_imaged_topology(
             [self.host_topology, mol],
             x0,
             box0,
-            "initial_"+prefix+".pdb"
+            f"initial_{prefix}.pdb"
         )
 
         for lambda_idx, res in enumerate(results):
-            # used for debugging for now, try to reproduce mdtraj error
-            # outfile = open("pickle_"+prefix+"_lambda_idx_" + str(lambda_idx) + ".pkl", "wb")
-            # pickle.dump((res.xs, res.boxes, combined_topology), outfile)
-            traj = mdtraj.Trajectory(res.xs, mdtraj.Topology.from_openmm(combined_topology))
-            traj.unitcell_vectors = res.boxes
-            traj.save_xtc("initial_"+prefix+"_lambda_idx_" + str(lambda_idx) + ".xtc")
+            np.savez(
+                f"initial_{prefix}_lambda_idx_{lambda_idx}.npz",
+                xs=res.xs,
+                boxes=res.boxes,
```

Comment:
Quick clarification: Are these in float32 or float64? (either is fine)

Reply:
These are currently in float32